include/net/buf.h

```diff
@@ -1523,7 +1544,10 @@ static inline void *net_buf_user_data(const struct net_buf *buf)
  * @param buf Buffer to update.
  * @param val 24-bit value to be added.
  */
-#define net_buf_add_le24(buf, val) net_buf_simple_add_le24(&(buf)->b, val)
+static inline void net_buf_add_le24(struct net_buf *buf, uint32_t val)
```

Comment:
You'll need to update the `@def net_buf_add_le24` in the doxygen documentation above (in fact, I think you can just remove it since it shouldn't be needed for functions). Same applies for all the conversions you're doing.

Reply:
Removed from all